LICENSE.txt

```diff
@@ -0,0 +1,177 @@
+
```

Comment:
TODO: Should be deleted.

Reply:
https://github.com/jenkinsci/workflow-cps-plugin/pull/612/commits/33e276c2c53e7178d7a81afb2db15aaf42393483